packages/hooks/README.md

```diff
@@ -99,3 +99,28 @@ Hook to subscribe to changes in viewport size
 ```js
 const viewportSize = useViewportSize();
 ```
+
+## usePoller
+
+Hook to create a Poller that polls at a given interval.
+
+If your `onPoll` handler returns a `Promise` it will wait for the Promise to resolve or reject before scheduling the next interval.
```

Comment:
Is it worth noting the behavior when the window isn't active?

Reply:
Yeah wasn't sure where to put that. Here works.